src/core/crypto.c

```diff
@@ -2542,3 +2542,88 @@ QuicCryptoDecodeClientTicket(
 Error:
     return Status;
 }
+
+QUIC_STATUS
+QuicCryptoReNegotiateAlpn(
+    _In_opt_ QUIC_CONNECTION* Connection,
+    _In_ uint16_t AlpnListLength,
+    _In_reads_bytes_(AlpnListLength)
+        const uint8_t* AlpnList
+    )
+{
+    if(Connection == NULL) {
```

Comment:
```suggestion
    if (Connection == NULL) {
```

Reply:
Actually CXPLAT_DBG_ASSERT that Connection and AlpnList are non-null. And that length is not zero.

Reply:
Done